mathgenerator/__init__.py

```diff
@@ -29,3 +29,46 @@ def getGenList():
 
 def genById(id, *args, **kwargs):
     return globals()[gen_list[id][0]](*args, **kwargs)
+
+def generate_context(seed = None):
+    if seed is not None:
+        random.seed(seed)
+    choice = random.choice(gen_list)
+    _ = globals()[choice[0]]()
+    problem, solution = _[0], _[1]
+    if len(_) == 3:
+        forward_words = _[2]
+    else:
+        forward_words = []
+    forward_words.extend(choice[0].split('_')).extend(choice[1].split('_'))
+    problem = str(problem)
+    solution = str(solution)
+    try:
+        # See if the solution is a float
+        float(solution.replace('$', ''))
+        return {
+            "problem": problem,
```

Comment:
not sure if it's the best design to outsource the context object to third party libaries. it will make testing and version control tricky, plus i've been redesigning the context objects so we may be stepping on each other's toes here.

since most fields are always the same we can probably do something universal for these different cases